### DISCUSSION TITLE
Ability to disable chat history

### DISCUSSION THREAD

**Discussion by badcom** [danny-avila/LibreChat#1694] [Feature Requests & Suggestions]
Apologies if this has already been raised elsewhere.

It would be good if there was an option to disable chat history for privacy and security reasons. I know that there's an option to clear all chats, but it would be helpful if there was an option to disable chats too.

Some companies may have concerns around this.

**Comment by danny-avila** [comment]
I'm not sure I understand this request. Having a history of chats is integral to the app. What's wrong with clearing/deleting conversations as is? That essentially handles your concern (all messages associated with the conversations are wiped).

**Reply by badcom** [reply]
New messages would exist until you don't close your browser. The users wouldn't have to clear / delete the messages all the time.

Having said that, given that there's an option to delete specific messages, the users could simply delete any messages that they deem confidential.

Scratch this idea =)
